test/test_core.jl

```diff
@@ -257,6 +261,18 @@ end
     @test get([4,5,6,7], l) == [4,5,6]
 end
 
+@testset "DynamicIndexLens" begin
+    l = @lens _[end]
+    obj = (1,2,3)
+    @test get(obj, l) == 3
+    @test set(obj, l, true) == (1,2,true)
+
+    l = @lens _[end÷2]
+    obj = (1,2,3)
+    @test get(obj, l) == 1
+    @test set(obj, l, true) == (true,2,3)
```

Comment:
Can you add some more complicated lenses? Composed and multi index? Also can you add `show` tests?

Reply:
OK. I added a few more tests. I completed forgot about multi-index version so that's implemented/tested as well.

Reply:
e.g. `@lens _[end, end]`